packages/api/src/client/index.ts

```diff
@@ -98,6 +98,11 @@ export * as AppBskyProfile from './types/app/bsky/profile'
 export * as AppBskyRepost from './types/app/bsky/repost'
 export * as AppBskyUpdateProfile from './types/app/bsky/updateProfile'
 
+export const APP_BSKY = {
+  ActorScene: 'app.bsky.actorScene',
+  ActorUser: 'app.bsky.actorUser',
+}
```

Comment:
_Yes_

Reply:
The tiger
He destroyed his cage
Yes
YES
The tiger is out

Reply:
hahaha incredible